src/sql/orig_results.sql

```diff
@@ -0,0 +1,15 @@
+CREATE VIEW total_counts AS
+  SELECT token, count(token) as count
+  FROM sim
+  GROUP BY token
+  ORDER BY count DESC;
+
+SELECT
```

Comment:
for each token, find the percentage of times it is >0.9 similar to its own token mean? 

i.e. how often are instances of "old" similar to mean("old")?

Reply:
im not sure what you mean by "how often are instances of "old" similar to mean("old")?". This sql file is the same as our results table we used for the preliminary results, the logic is just in SQL instead of python... we will want to run a few more queries for the final results.

Reply:
right but instead of comparing similarity to the mean of all hidden states of all risk factors, we're comparing how similar instances of a risk factor are to the mean of its own hidden states right? that's what I'm a little confused abt